plugins/coral-plugin-love/client/config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "Coral Plugin Love",
+  "slot": "Comment.Detail"
```

Comment:
Comment is a Class and a Component, that's the reason behind it. Not strongly against it either. We can change it.